sys/riscv/riscv/machdep.c

```diff
@@ -449,7 +449,7 @@ fake_preload_metadata(struct riscv_bootparams *rvbp)
 	/* Copy the DTB to KVA space. */
 	dtb_size = fdt_totalsize(rvbp->dtbp_virt);
 #ifdef __CHERI_PURE_CAPABILITY__
```

Comment:
Can drop the ifdef as well? And maybe move it after the first align?

Reply:
Dropping the #ifdef is fine.  I think the ALIGN_UP needs to be first as that is about the padding at the end of the DTB.  The other roundup is to make sure the following PRELOAD_PUSH_VALUE is aligned, so I think the order should stay as it is.